tasks/testChainAdapter.ts

```diff
@@ -0,0 +1,68 @@
+import assert from "assert";
+import { getMnemonic } from "@uma/common";
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { CHAIN_IDs, TOKEN_SYMBOLS_MAP } from "../utils/constants";
+
+// Chain adapter names are not 1:1 consistent with chain names, so some overrides are needed.
+const chains = {
+  [CHAIN_IDs.ARBITRUM]: "Arbitrum_Adapter",
+  [CHAIN_IDs.WORLD_CHAIN]: "WorldChain_Adapter",
+  [CHAIN_IDs.ZK_SYNC]: "ZkSync_Adapter",
+};
+
+task("testChainAdapter", "Verify a chain adapter")
+  .addParam("chain", "chain ID of the adapter being tested")
+  .addParam("token", "Token to bridge to the destination chain")
```

Comment:
Should probably `addParam("amount", ...)` here to allow the user to specify the amount to bridge.

Reply:
https://github.com/across-protocol/contracts/pull/669/commits/8cae0b6fa568e3ece12d4ab6f510019af86aadb9